company/team/nick.md

```diff
@@ -0,0 +1,32 @@
+# Nick Snyder
+
+## 1-1s
+
+_I will be experimenting with a modified 1-1 format, as documented below. The goal is to have more frequent, but shorter 1-1s so longer discussions are short circuited and discussed async._
+
+This is how I run [1-1s](../../handbook/leadership/1-1.md) with my direct reports.
+
+- 1-1s are 25 minutes weekly.
+- 1-1s start 5 minutes after the hour or half hour (e.g. 9:05, 9:35).
+- 5 minutes before the 1-1 starts (e.g. 9:00, 9:30), I will write a private list of the most important things I want to discuss in priority order. Direct reports should do the same.
+- Agenda:
+  - Say hello and chit-chat about personal lives (<5 minutes).
+  - Paste each persons agenda items into the [shared Google Doc](../../handbook/leadership/1-1.md#google-doc).
```

Comment:
Curious: do you see keeping the agenda items private until the beginning of the 1:1 as a hard requirement, and if so, can you document the rationale behind it? I have often written down agenda items in our shared 1:1 doc ahead of time.

Reply:
Keeping them private achieves the property that the lists are generated independently. The independence means that items on both lists are high signal and also you will discover what things are important to you but not the other person.

I agree that it is slightly more inconvenient to not be able to drop notes directly into the shared 1-1 doc. As a compromise, I can achieve the goals above completely on my end by just generating my list independently before looking at the shared doc. This will allow direct reports to continue to use the shared doc.

I removed the "private" wording in dcd5897a08d42ab4b896f42941fbad57e9f8b536

Reply:
Makes sense, thanks for the clarification!